### PR TITLE
chore: remove Argo CD CRDs from namespaced install

### PR DESCRIPTION
**Pull request by alexmt**
Signed-off-by: Alexander Matyushentsev <AMatyushentsev@gmail.com>

The namespaced install is used by Argo CD admins who manage multiple Argo CD instances within one namespace. As per [description](https://github.com/argoproj/argo-cd/tree/master/manifests), it does not require any admin permissions. CRDs are cluster lever resources so should be removed from namespaced manifests bundle.

### CODE REVIEW

**Comment by alexmt** [comment]
We use namespaced bundle to manage multiple Argo CD instances deployed into separate namespaces. Because CRDs are bundled we cannot just reference `namespace-install.yaml` in kustomize.yaml and instead have to copy it and remove CRDs section. 

I think other users have the same use case . @sbose78 , @jannfis what do you think?

File: manifests/README.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ Four sets of installation manifests are provided:
   external clusters. Will still be possible to deploy to the same cluster (kubernetes.svc.default)
   with inputted credentials (i.e. `argocd cluster add <CONTEXT> --in-cluster --namespace <YOUR NAMESPACE>`).
 
+> Note: Argo CD CRDs are not included into [namespace-install.yaml](namespace-install.yaml) and have to be installed
+> separately.
```
Comment (jannfis) [inline manifests/README.md:21]: Can we give instructions on *how* to install the CRDs manually? E.g. where to find them in the code base?

**Review by jannfis — COMMENTED** [review]
Indeed, the namespaced install probably should not include the CRDs. I have just a minor comment, no formal request for change. Please see below.

File: manifests/README.md (hunk 1 above)
Comment (alexmt) [reply manifests/README.md]: good point. added

**Comment by sbose78** [comment]
Not a big deal, but  Argo CD wouldn't be useful without the CRDs :(

So, the admin anyway needs to have permissions to create CRDs at some
point, just that everything else would be namespace scoped :)

Do you foresee someone else creating the CRDs?

**Comment by kshamajain99** [comment]
@alexmt we should make respective changes in `manifests/ha/namespace-install` as well
https://github.com/argoproj/argo-cd/tree/master/manifests/ha/namespace-install

**Comment by alexmt** [comment]
Thank you @kshamajain99 ! Fixed.

**Comment by alexmt** [comment]
@sbose78, CRDs have to be applied by cluster-admin, not namespace admin. For example, in our case, CRDs and all other cluster-level resources are managed by a platform team. 

Our team is responsible for managing Argo CD instances (each instance in a separate namespace). If we removing CRD from namespace-install.yaml simplifies the process for us. We can just reference `namespace-install.yaml` in kustomization.yaml instead of copying to modifying it.

**Review by kshamajain99 — APPROVED** [review]

**Pull request merged**